src/dev/src/app/components/tree-view/tree-view-client-side-demo.ts

```diff
@@ -390,7 +416,12 @@ export class TreeViewClientSideDemo {
           return { data: it, nodeLabel: it.name };
         })
       );
-    },
+    }
+  };
+  treeViewAutoSelector: TreeViewAutoSelector<FoodNode> = {
```

Comment:
Just kee this one only. We don't need two autoSelector. And please rename it into `treeViewAutoNodeSelector`.

Reply:
Done